vpr/src/place/place.cpp

```diff
@@ -1290,14 +1451,16 @@ static bool update_annealing_state(t_annealing_state* state,
     return true;
 }
 
-static float starting_t(t_placer_costs* costs,
+static float starting_t(float crit_exponent,
+                        t_placer_costs* costs,
                         t_placer_prev_inverse_costs* prev_inverse_costs,
                         t_annealing_sched annealing_sched,
                         int max_moves,
                         float rlim,
                         const PlaceDelayModel* delay_model,
-                        const PlacerCriticalities* criticalities,
-                        TimingInfo* timing_info,
+                        PlacerCriticalities* criticalities,
+                        PlacerSetupSlacks* setup_slacks,
```

Comment:
`criticalities` and `setup_slacks` should be bundled together because they used together.

Reply:
I want to do this in another PR, where I will refactor place.cpp. But if you want it done here, I will implement it. On a side note, how would you prefer naming this new structure?